src/test/resources/logging.properties

```diff
@@ -0,0 +1,10 @@
+# This is the default java.util.logging.config.file the Maven Surefire plugin will use (see pom.xml for configuration)
```

Comment:
What does this file do that `src/test/resources/log4j2.xml` cannot do?

Reply:
`src/test/resources/log4j2.xml` isn't able to affect any logging that occurs through `java.util.logging`.  Commonly used dependencies can use that package to log their information and without this configuration file, it can be frustrating to find a way to change what is being logged by those dependencies.

Reply:
Good point, that probably needs to be explained in the comments for the file, so future developers don't get confused.

Reply:
Done: https://github.com/reactome/maven-project-template/pull/1/commits/43f193f329074dc136edac4630db64727db96d8a